docs/src/app/components/pages/components/icon-menus.jsx

```diff
@@ -71,6 +71,12 @@ export default class IconMenus extends React.Component {
             desc: 'If true, menu will close after an item is touchTapped.',
           },
           {
+            name: 'open',
+            type: 'bool',
+            header: 'default: true',
```

Comment:
> If you mean this is optional prop

Yes, I think that the open value should be optional. When it's provided,
the component behaves as a controlled one and called `onRequestChange`.
When it's not provided, the component should behave as before.

So, I don't really know what is best between `null` or `undefined`.
@shaurya947 What do you think?